regulations/static/regulations/css/less/module/comment-confirm.less

```diff
@@ -0,0 +1,42 @@
+
+.comment-confirm-success {
+  .font-regular;
+  padding: 30px 20px;
+  width: 80%;
+
+  .success-alert {
+    background-color: @green_light;
+    display: -webkit-flex;
+    display: flex;
```

Comment:
I did a check in Virtualbox with IE11 and it loads okay. According to ATF analytics only 24% are on IE, and of that 87% are on IE11.